unlock-protocol.com/next.config.js

```diff
@@ -10,16 +10,17 @@ const { addBlogPagesToPageObject } = require('./src/utils/blog')
 const copyFile = promisify(fs.copyFile)
 
 const unlockEnv = process.env.UNLOCK_ENV || 'dev'
+const googleAnalyticsId = process.env.UNLOCK_GA_ID || '0'
```

Comment:
how would you feel about us setting up a staging id to ensure that we don't lose anything to the 0?

Reply:
I think that totally makes sense!